data.sql

```diff
@@ -1,5 +1,5 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+insert into animals values (1, 'Agumon', '2020-02-03', 0, true, 10.23);
+insert into animals values (2, 'Gabumon', '2018-11-15', 2, true, 8.0);
+insert into animals values (3, 'Pikachu', '2021-01-7', 1, false, 15.04);
+insert into animals values (4, 'Devimon', '2017-05-12', 5, true, 11.0);
```

Comment:
- [ ] I notice that you have not added code that addresses the questions below: 

- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 
**Kindly add these codes and attach screenshots of the results of your queries.**
Please label each screenshot with the question. I see that you added some screenshots, please add labels.